cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -81,227 +74,246 @@ CREATE CONSTRAINT FOR ... REQUIRE ...
 [source, cypher, role="noheader"]
 ----
-CREATE RANGE INDEX ...
+point.withinBBox(point({x:1, y:1}), point({x:0, y:0}), point({x:2, y:2}))
 ----
 
-These new indexes may be combined for multiple use cases.
-
 a|
 label:functionality[]
 label:deprecated[]
+label:new[]
 [source, cypher, role="noheader"]
 ----
-CREATE INDEX
-...
-OPTIONS "{" btree-option: btree-value[, ...] "}"
+DROP CONSTRAINT
+ON (n:LabelName)
+ASSERT (n.propertyName_1, …, n.propertyName_n) IS UNIQUE
+----
+a|
+Unique property constraints now allow multiple properties.
+
+Replaced by:
+[source, cypher, role="noheader"]
 ----
+DROP CONSTRAINT name [IF EXISTS]
+----
+
+|===
 
+=== New features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
 
 a|
-label:functionality[]
-label:deprecated[]
+label:syntax[]
+label:new[]
 [source, cypher, role="noheader"]
 ----
-SHOW BTREE INDEXES
+CREATE CONSTRAINT FOR ... REQUIRE ...
 ----
 a|
-B-tree indexes are deprecated, partially replaced for now, and will be fully replaced in 5.0 by <<indexes-future-indexes, future indexes>>.
-In 4.4, b-tree indexes are still the correct alternative to use.
+New syntax for creating constraints, applicable to all constraint types.
 
-Replaced by the new and future index types:
+
+a|
+label:syntax[]
+label:new[]
 [source, cypher, role="noheader"]
 ----
-SHOW {POINT \| RANGE \| TEXT} INDEXES
+CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
+FOR ...
+REQUIRE ... IS NOT NULL
+OPTIONS "{" "}"
 ----
+a|
+Existence constraints now allow an `OPTIONS` map, however, at this point there are no available values for the map.
 
 
 a|
 label:functionality[]
-label:deprecated[]
+label:new[]
 [source, cypher, role="noheader"]
 ----
-CREATE CONSTRAINT
-...
-OPTIONS "{" btree-option: btree-value[, ...] "}"
+CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+REQUIRE (n.propertyName_1, …, n.propertyName_n) IS UNIQUE
+[OPTIONS "{" option: value[, ...] "}"]
 ----
 a|
-Node key and uniqueness constraints with b-tree options are deprecated and will be replaced in 5.0 by range options, see <<indexes-future-indexes, range indexes>>.
-In 4.4, the b-tree index-backed constraints are still the correct alternative to use.
+Unique property constraints now allow multiple properties, ensuring that the combination of property values are unique.
 
-Will be replaced by:
+a|
+label:functionality[]
+label:new[]
 [source, cypher, role="noheader"]
 ----
-CREATE CONSTRAINT
-...
-OPTIONS "{" range-option: range-value[, ...] "}"
+CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
+FOR ... ON ...
+OPTIONS "{" option: value[, ...] "}"
 ----
-Constraints used for string properties will also require an additional text index to cover the string queries properly.
-Constraints used for point properties will also require an additional point index to cover the spatial queries properly, see <<indexes-future-indexes, point indexes>>.
+a|
+Token lookup indexes now allow an `OPTIONS` map to specify the index provider.
```

Comment:
why change the order of these from before?

The one you placed first and last is more connected to the same topic while the one you put in the middle is different (and was first before)?

Before:
```
multi-property uniqueness constraints
options clause for existence constraints
options clause for lookup indexes
```
Now:
```
options clause for existence constraints
multi-property uniqueness constraints
options clause for lookup indexes
```